# This represents the parameters of ball_detector

uint32  scaling
float32 confidence
float32 mean_val
